crates/datasources/src/mysql/mod.rs

```diff
@@ -374,6 +375,54 @@ impl TableProvider for MysqlTableProvider {
             metrics: ExecutionPlanMetricsSet::new(),
         }))
     }
+
+    async fn insert_into(
+        &self,
+        state: &SessionState,
+        input: Arc<dyn ExecutionPlan>,
+        _overwrite: bool,
+    ) -> DatafusionResult<Arc<dyn ExecutionPlan>> {
+        let mut values = String::new();
+
+        let mut input = execute_stream(input, state.task_ctx())?;
+        while let Some(batch) = input.next().await {
+            let batch = batch?;
+
+            let mut row_sep = "";
+            for row_idx in 0..batch.num_rows() {
+                write!(&mut values, "{row_sep}(")?;
+                row_sep = ",\n\t";
```

Comment:
do we really need to have these new-lines and indentation?

It seems like a lot of code to add new lines and what not to statements where we're just going to pass it to another service, where either no one will appreciate the pretty formatting, or it will show up in logs where multi-line logging isn't helpful

Reply:
Yeah, I added new lines for debugging. Helps on console!

Reply:
Given that people don't see it, except in logs, (and again, multi-line logging is going to be confusing in production) I think we shouldn't prioritize the new lines?

Reply:
Yeah, we can remove new lines here.

Reply:
I have removed the tabs and new lines